PetAdoption-iOS/PetAdoption-iOS/Extensions/UIImage+Placeholder.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UIImage+Placeholder.swift
+//  PetAdoption-iOS
+//
+//  Created by Keli'i Martin on 8/20/16.
+//  Copyright © 2016 Code For Orlando. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+import AlamofireImage
+
+extension UIImage
```

Comment:
This is probably not the greatest named extension...  It actually just lets me load images from URLs async...